### PR TITLE
7038 book a demo page

### PR DESCRIPTION
**Pull request by SeanPackham**

### CODE REVIEW

File: source/demo.html.erb [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          <p>
+            <small>
+              In the meantime, why not sign up for a free 30 day trial or learn
+              more about our Technology?
+            </small>
+          </p>
+        </div>
+        <div class="social">
+          <h3>Follow us</h3>
+          <a target="_blank" href="https://www.facebook.com/CodioHQ" class="icon-10 facebook" title="Facebook"><svg viewBox="0 0 512 512"><path d="M211.9 197.4h-36.7v59.9h36.7V433.1h70.5V256.5h49.2l5.2-59.1h-54.4c0 0 0-22.1 0-33.7 0-13.9 2.8-19.5 16.3-19.5 10.9 0 38.2 0 38.2 0V82.9c0 0-40.2 0-48.8 0 -52.5 0-76.1 23.1-76.1 67.3C211.9 188.8 211.9 197.4 211.9 197.4z"/></svg><!--[if lt IE 9]><em>Facebook</em><![endif]--></a>
```
Comment (Dignifiedquire) [inline source/demo.html.erb]: Please put the svg images into individual files and use middleman to inline them when building

File: source/stylesheets/pages/_demo.scss [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+          height: 100%;
+          width: 100%;
+          color: #fff;
+          fill: #fff;
+        }
+      }
+    }
+  }
+
+
```
Comment (Dignifiedquire) [inline source/stylesheets/pages/_demo.scss]: bit much whitespace ;)

File: source/stylesheets/pages/_demo.scss [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+  @include breakpoint($max-width) {
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  h1 {
+    margin-top: 4 * $base-line-height;
+    font-size: 52px;
+    font-weight: 400;
+    // text-align: center;
```
Comment (joelmoss) [inline source/stylesheets/pages/_demo.scss]: Please remove commented out code if not needed

File: source/stylesheets/pages/_demo.scss [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+      width: 100%;
+    }
+
+    span.required {
+      content:" *";
+      width: 1em;
+      height: 40px;
+      position: relative;
+      left: -1rem;
+      // border-color: red !important;
```
Comment (joelmoss) [inline source/stylesheets/pages/_demo.scss]: Please remove commented out code if not needed

File: source/stylesheets/pages/_demo.scss [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+      padding: 0 2px;
+      margin-right: 10px;
+      text-align: center;
+    }
+
+    .checkbox-custom:checked + .checkbox-custom-label:before {
+      color: #FFFFFF;
+      content:'\2713';
+      // background: rebeccapurple;
+      // box-shadow: inset 0px 0px 0px 4px #fff;
```
Comment (joelmoss) [inline source/stylesheets/pages/_demo.scss]: Please remove commented out code if not needed

File: source/stylesheets/pages/_demo.scss [hunk 6]
```diff
@@ -0,0 +1,304 @@
...
+
+      .social {
+        border-top: 1px solid #7a9fe5;
+
+        a, svg {
+          display: inline-block;
+        }
+
+        a {
+          // position: relative; // IE8
```
Comment (joelmoss) [inline source/stylesheets/pages/_demo.scss]: Please remove commented out code if not needed

File: source/demo.html.erb [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+          <p>
+            <small>
+              In the meantime, why not sign up for a free 30 day trial or learn
+              more about our Technology?
+            </small>
+          </p>
+        </div>
+        <div class="social">
+          <h3>Follow us</h3>
+          <a target="_blank" href="https://www.facebook.com/CodioHQ" class="icon-10 facebook" title="Facebook">
```
Comment (Dignifiedquire) [inline source/demo.html.erb]: Please use the `link_to` helper for links, like here: https://github.com/codio/site/blob/master/source/partials/_footer.erb#L59

File: source/demo.html.erb [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+            <%= inline_svg "assets/demo/facebook.svg", class: "" %>
+            <!--[if lt IE 9]><em>Facebook</em><![endif]-->
+          </a>
+          <a target="_blank" href="https://twitter.com/CodioHQ" class="icon-26 twitter" title="Twitter">
```
Comment (Dignifiedquire) [inline source/demo.html.erb]: `link_to`

File: source/demo.html.erb [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+            <%= inline_svg "assets/demo/twitter.svg", class: "" %>
+            <!--[if lt IE 9]><em>Twitter</em><![endif]-->
+          </a>
+          <a target="_blank" href="https://www.linkedin.com/company/codio" class="icon-17 linkedin" title="LinkedIn">
```
Comment (Dignifiedquire) [inline source/demo.html.erb]: `link_to`

File: source/demo.html.erb [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+              In the meantime, why not sign up for a free 30 day trial or learn
+              more about our Technology?
+            </small>
+          </p>
+        </div>
+        <div class="social">
+          <h3>Follow us</h3>
+          <a target="_blank" href="https://www.facebook.com/CodioHQ" class="icon-10 facebook" title="Facebook">
+            <%= inline_svg "assets/demo/facebook.svg", class: "" %>
+            <!--[if lt IE 9]><em>Facebook</em><![endif]-->
```
Comment (Dignifiedquire) [inline source/demo.html.erb]: We actually do not support anything below IE 10, so not sure we need this. cc @joelmoss

Comment (SeanPackham) [inline source/demo.html.erb]: @Dignifiedquire should I remove the fallback them?

**Comment by Dignifiedquire** [comment]
:+1:

**Pull request merged**